go.mod

```diff
@@ -13,5 +13,6 @@ require (
 	github.com/davecgh/go-spew v1.1.0 // indirect
 	github.com/philhofer/fwd v1.1.1 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
+	golang.org/x/text v0.3.7 // indirect
```

Comment:
It seems silly to me to pull in a new library just to avoid a bug in the stdlib that doesn't actually affect our code; none of these names have unicode punctuation in them, and I don't really forsee that changing, so this seems like it's just going to bloat binaries with no upside? Haven't looked into the implementation, but particiularly since we're not doing a lot of text processing elsewhere in the repo, I wouldn't be surprised if this ended up pulling in a bunch of large unicode tables we aren't otherwise using.

Reply:
Fair point.  I'll revert this now.